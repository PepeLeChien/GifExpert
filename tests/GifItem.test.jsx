import { render } from "@testing-library/react"
import { GifItem } from "../src/components/GifItem"

describe('tests on GifItem', () => {

    test('should match with snapshot', () => { 
        
        const {container} = render(<GifItem title="One Punch" url="https://blablabla.com"/>);

        expect(container).toMatchSnapshot();
    })
})