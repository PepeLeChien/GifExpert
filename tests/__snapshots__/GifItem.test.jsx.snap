// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests on GifItem should match with snapshot 1`] = `
<div>
  <div
    class="card"
  >
    <img
      alt="One Punch"
      src="https://blablabla.com"
    />
    <p>
      One Punch
    </p>
  </div>
</div>
`;
